src/dmd/expressionsem.d

```diff
@@ -2326,8 +2326,8 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
 
         /* Look for what user might have meant
          */
-        if (const n = importHint(exp.ident.toChars()))
-            exp.error("`%s` is not defined, perhaps `import %s;` is needed?", exp.ident.toChars(), n);
+        if (const n = importHint(exp.ident.toString()))
+            exp.error("`%s` is not defined, perhaps `import %s;` is needed?", exp.ident.toChars(), n.ptr);
```

Comment:
Isn't it better to use `%.*` and pass the length and pointer separately?

Reply:
Was taking advantage of that strings from importHint are null terminated.  Can do though.

Reply:
I’m guessing it won’t be null terminated after most code uses D strings.